chain-signatures/node/src/metrics.rs

```diff
@@ -389,6 +389,56 @@ pub(crate) static SIGNATURE_PUBLISH_FAILURES: Lazy<CounterVec> = Lazy::new(|| {
     .unwrap()
 });
 
+// CPU Usage Percentage Metric
+pub(crate) static CPU_USAGE_PERCENTAGE: Lazy<IntGaugeVec> = Lazy::new(|| {
+    try_create_int_gauge_vec(
+        "multichain_cpu_usage_percentage",
+        "CPU Usage Percentage",
+        &["node_account_id"],
+    )
+    .unwrap()
+});
+
+// Total Memory Metric
+pub(crate) static TOTAL_MEMORY_BYTES: Lazy<IntGaugeVec> = Lazy::new(|| {
```

Comment:
Either this one or `AVAILABLE_MEMORY_BYTES` is redundant.

Reply:
Didn't we want a total to ensure partners have enough memory? I figured available would be useful to alert off of as well.

Reply:
Just saying that available + used = total